test/functional/admin/organisations_controller_test.rb

```diff
@@ -83,7 +83,7 @@ def example_organisation_attributes
   test 'POST :create can set a custom logo' do
     post :create, organisation: example_organisation_attributes.merge(
       organisation_logo_type_id: OrganisationLogoType::CustomLogo.id,
-      logo: fixture_file_upload('logo.png')
+      logo: fixture_file_upload('logo.png', 'image/png')
```

Comment:
If you don't provide it `content_type` returns `nil` and I needed to check it to decide whether to do the thumbnails or not.